content/docs/setup/kubernetes/quick-start-alicloud-ack/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Quick Start with AliCloud Kubernetes Container Service
+description: Quick Start instructions to setup the Istio service using AliCloud Kubernetes Container Service
+weight: 12
+keywords: [kubernetes,alicloud]
+---
+
+Quick Start instructions to install and run Istio in [AliCloud Kubernetes Container Service](https://cs.console.aliyun.com/) using Application Catalog.
+
+This Quick Start installs the current release version of Istio and then deploys the [Bookinfo](/docs/examples/bookinfo/) sample
+application.  It uses Application Catalog to automate the steps detailed in the [Istio on Kubernetes setup guide](/docs/setup/kubernetes/quick-start/) for Kubernetes Engine.
+
+## Prerequisites
+
+- This sample assumes that you already have an avaiable AliCloud Kubernetes cluster. Otherwise, you can create a Kubernetes cluster quickly and easily in the Container Service console.
+
+- Make sure that `kubectl` can work fine for your Kubernetes cluster.
+
+- Create a namespace to deploy Istio components, e.g. `istio-system`:
+
+{{< text bash >}}
```

Comment:
this should be removed and linked to the helm installation documentation so it can be maintained in only one place.

Reply:
@geeknoid what is our recommendation for adding a dedicated page for a cloud provider vs having it under the general Quick Start?  Should we move each cloud provider's quick start to its own sub pages?

Reply:
Yep, we're on the subpage route which I think scales well.